### PR TITLE
update-2023-end

### PR DESCRIPTION
**Pull request by Damika-Anupama**
update couple of files according to the latest aws version. Tested sam build and deploy, checked the apis, lambdas, tables

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -13,12 +13,12 @@ build-writeItemFunction:
...
+	printf "{\"extends\": \"./tsconfig.json\", \"include\": [\"${HANDLER}\"] }" > tsconfig-only-handler.json
 	npm run build -- --build tsconfig-only-handler.json
 	cp -r dist "$(ARTIFACTS_DIR)/"
 
 build-RuntimeDependenciesLayer:
 	mkdir -p "$(ARTIFACTS_DIR)/nodejs"
 	cp package.json package-lock.json "$(ARTIFACTS_DIR)/nodejs/"
 	npm install --production --prefix "$(ARTIFACTS_DIR)/nodejs/"
-	rm "$(ARTIFACTS_DIR)/nodejs/package.json" # to avoid rebuilding when changes doesn't relate to dependencies
+	rm "$(ARTIFACTS_DIR)/nodejs/package.json"
```
Comment (Envek) [inline Makefile:24]: Can we please keep original comment?
```suggestion
	rm "$(ARTIFACTS_DIR)/nodejs/package.json" # to avoid rebuilding when changes doesn't relate to dependencies
```

File: Makefile [hunk 2]
```diff
@@ -13,12 +13,12 @@ build-writeItemFunction:
 build-lambda-common:
 	npm install
 	rm -rf dist
-	echo "{\"extends\": \"./tsconfig.json\", \"include\": [\"${HANDLER}\"] }" > tsconfig-only-handler.json
+	printf "{\"extends\": \"./tsconfig.json\", \"include\": [\"${HANDLER}\"] }" > tsconfig-only-handler.json
```
Comment (Envek) [inline Makefile:16]: What is the reason for this change?

**Review by Envek — CHANGES_REQUESTED** [review]
Thanks for the pull request! Just some minor comments

**Comment by Damika-Anupama** [comment]
Hi I used WSL, Makefile gave an error with the comment, that's why I removed it.

File: Makefile (hunk 2 above)
Comment (Damika-Anupama) [reply Makefile:16]: WSL didn't recognize echo, problem solved with printf
